HousingSearchApi/data/elasticsearch/assetIndex.json

```diff
@@ -1,109 +1,74 @@
 {
-  "aliases": {
-
-  },
   "mappings": {
     "properties": {
       "id": {
-        "type": "text",
-        "fields": {
-          "keyword": {
-            "type": "keyword",
-            "ignore_above": 256
-          }
-        }
+        "type": "keyword"
       },
       "assetId": {
-        "type": "text",
-        "fields": {
-          "keyword": {
-            "type": "keyword",
-            "ignore_above": 256
-          }
-        }
+        "type": "keyword"
       },
       "assetType": {
         "type": "text",
         "fields": {
           "keyword": {
-            "type": "keyword",
-            "ignore_above": 256
+            "type": "keyword"
           }
         }
       },
-      "parentAssetIds": {
-        "type": "text",
-        "fields": {
-          "keyword": {
-            "type": "keyword",
-            "ignore_above": 256
-          }
-        }
+      "isAssetCautionaryAlerted": {
+        "type": "boolean"
       },
       "assetAddress": {
         "properties": {
           "uprn": {
             "type": "keyword",
-            "normalizer": "my_normalizer",
-            "fields": {
-              "keyword": {
-                "type": "keyword",
-                "ignore_above": 256
-              }
-            }
+            "normalizer": "my_normalizer"
           },
-          "postPreamble": {
+          "addressLine1": {
```

Comment:
Key change 2: `addressLine1` is primarily text (good for partial/fuzzy matching) instead of keyword (good for exact matching)